drivers/pinmux/stm32/pinmux_stm32f1.h

```diff
@@ -53,6 +53,9 @@
 #define STM32F1_PINMUX_FUNC_PC10_UART4_TX            STM32_PIN_USART_TX
 #define STM32F1_PINMUX_FUNC_PC11_UART4_RX            STM32_PIN_USART_RX
 
+#define STM32F1_PINMUX_FUNC_PC12_UART5_TX            STM32_PIN_USART_TX
```

Comment:
I'm curious about this uart5, I'm not seeing any u(s)art5 instance on STM32F1.
Can you have a new check?